nixos/modules/services/web-apps/plausible.md

```diff
@@ -15,15 +15,6 @@ After that, `plausible` can be deployed like this:
 {
   services.plausible = {
     enable = true;
-    adminUser = {
```

Comment:
This may warrant a bit of text in the release notes. User may find it surprising that their NixOS config doesn't evaluate anymore and it's not easy to find out what happened.